src/components/form/contacts/index.tsx

```diff
@@ -40,18 +57,59 @@ const ContactsForm = ({ toggleModal, initialValues, isEdit, apiUrl }: IProps) =>
         await dispatch(updateContactAsync({ id: body.id, data: body }));
       } else {
         const body = { ...data } as IContacts;
-        await dispatch(createContactAsync({ apiUrl, data: body }));
+        await dispatch(createContactAsync({ apiUrl, data: body, user: userStore }));
       }
       toggleModal();
     } catch (error) {
       Notify('Something went wrong');
     }
   };
 
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const fileInput = event.target;
+    if (fileInput.files && fileInput.files.length > 0) {
+      setFileName(fileInput.files[0].name);
+    } else {
+      setFileName(null);
+    }
+  };
+
+  const getHelperComponent = () => {
```

Comment:
first of all we can't use  switch case in this way. the main ideaa is to check some condition for one field.
And we dond need validation for img. We can create a contact without img.

Reply:
fix: all logic removed. validation for img is not required